src/service/firebaseConfig.ts

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from 'firebase/app';
+import { getFirestore } from 'firebase/firestore';
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: '$API_KEY',
+  authDomain: '$PROJECT_ID.firebaseapp.com',
+  projectId: '$PROJECT_ID',
+  storageBucket: '$PROJECT_ID.appspot.com',
+  messagingSenderId: '$MESSAGING_SENDER_ID',
+  appId: '$APP_ID',
+  measurementId: '$MEASUREMENT_ID'
+};
```

Comment:
Perhaps we should use .env variables otherwise I will be the next one who'll commit the credentials 🤣 
btw how will it work in production without a valid configuration?

Reply:
yeah, I'll need to work a bit on github-pages environment I guess

Reply:
@Undermaken I've just tested and the website will work, when you try to send feedback it will go in error showing the alert. I think it's fine for us, until we find a way to add environment vars in deployment

Reply:
Sure!
Let's go!